contracts/storage/AskStorage.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.20;
+
+import {ShardingTableStorage} from "./ShardingTableStorage.sol";
+import {ParametersStorage} from "./ParametersStorage.sol";
+import {ProfileStorage} from "./ProfileStorage.sol";
+import {StakingStorage} from "./StakingStorage.sol";
+import {INamed} from "../interfaces/INamed.sol";
+import {IVersioned} from "../interfaces/IVersioned.sol";
+import {IInitializable} from "../interfaces/IInitializable.sol";
+import {ContractStatus} from "../abstract/ContractStatus.sol";
+
+contract AskStorage is INamed, IVersioned, ContractStatus, IInitializable {
+    string private constant _NAME = "Ask";
+    string private constant _VERSION = "1.0.0";
+
+    uint256 public constant UPPER_BOUND_FACTOR = 1467000000000000000;
```

Comment:
how are these calculated ? Shouldn't there be a way to update them, maybe in parameters storage ?